source/legacy.txt

```diff
@@ -37,6 +39,26 @@ In addition to updating your application to handle any necessary changes,
 always check for any other differences in options and return values before
 moving it to production.
 
+Current API Benefits
+--------------------
+
+You can continue to use the legacy API with each new {+mdb-server+} release. 
+However, the legacy API does not support any new features introduced in {+mdb-server+}
+3.0 or later. Upgrade to the current API to use the following features:
```

Comment:
Let's not imply that this is an exhaustive list, as it is not.

Reply:
I changed the wording a bit so that it's more clear that this list is just a few examples.